source/code/source/playfab/PlayFabEventPipeline.cpp

```diff
@@ -110,6 +110,30 @@ namespace PlayFab
             auto playFabEmitEventResponse = std::make_shared<PlayFabEmitEventResponse>();
             playFabEmitEventResponse->emitEventResult = emitResult;
 
+            std::shared_ptr<PlayFabError> emitEventError  = std::make_shared<PlayFabError>();
+
+            emitEventError->ErrorName = "PlayFabEventPipeline IntakeEvent Error";
+            emitEventError->ErrorMessage = "PlayFabEventPipeline did not accept the event. Please see ErrorDetails for more information.";
```

Comment:
>"PlayFabEventPipeline did not accept the event. Please see ErrorDetails for more information."; [](start = 43, length = 95)

what's the point of this string if it just redirects you to ErrorDetails? why not just put the ErrorDetails string in both ErrorMessage and ErrorDetails? maybe there's some distinction between ErrorDetails and ErrorMessage I'm not seeing yet, but ErrorMessage seems pretty unhelpful here.

Reply:
The distinction comes into play when a user calls the GenerateErrorReport(). But I guess we can leave out directing users to the details then since it will just show up.